docs/configuration.md

```diff
@@ -85,7 +85,10 @@ Config file:
     "allowColor": true,
     "allowEmojis": false,
     "logTests": true,
-    "maxTestsToLog": 3
+    "maxTestsToLog": 3,
+    "reportAllTests": true,
+    "reportAllMutants": true,
```

Comment:
Could we rephrase them as `reportTests` and `reportMutants`? That way we could in the future expand the options here to specify states as an array. For example `"reportMutants": ["killed"]`

Reply:
Absolutely, those naming conventions do fit better especially for future-proofing for later expansion.